TestDockerPluginSuite/TestPluginInstallArgs  nil panic on s390x and ppc64le

https://ci-next.docker.com/public/blue/organizations/jenkins/moby/detail/master/487/pipeline/213

```
=== FAIL: s390x.integration-cli TestDockerPluginSuite/TestPluginInstallArgs (15.01s)
    --- FAIL: TestDockerPluginSuite/TestPluginInstallArgs (15.01s)
        suite.go:65: test suite panicked: runtime error: invalid memory address or nil pointer dereference
            goroutine 28144 [running]:
            runtime/debug.Stack(0xc0011c9618, 0x2649160, 0x3b1fd20)
            	/usr/local/go/src/runtime/debug/stack.go:24 +0x98
            github.com/docker/docker/internal/test/suite.failOnPanic(0xc000b8de00)
            	/go/src/github.com/docker/docker/internal/test/suite/suite.go:65 +0x5a
            panic(0x2649160, 0x3b1fd20)
            	/usr/local/go/src/runtime/panic.go:679 +0x1d4
            github.com/docker/docker/integration-cli.(*DockerPluginSuite).TestPluginInstallArgs.func1(0xc000bfd800)
            	/go/src/github.com/docker/docker/integration-cli/docker_cli_plugins_test.go:200 +0xba
            github.com/docker/docker/testutil/fixtures/plugin.CreateInRegistry(0x2d136a0, 0xc000bfd7a0, 0xc00098ad50, 0x2f, 0x0, 0xc0011c9a90, 0x1, 0x1, 0x0, 0x0)
            	/go/src/github.com/docker/docker/testutil/fixtures/plugin/plugin.go:95 +0x1a2
            github.com/docker/docker/integration-cli.(*DockerPluginSuite).TestPluginInstallArgs(0xc0007210b0, 0xc000b8de00)
            	/go/src/github.com/docker/docker/integration-cli/docker_cli_plugins_test.go:199 +0x122
            reflect.Value.call(0xc000357680, 0xc0000cb228, 0x13, 0x28ccff2, 0x4, 0xc000cddf40, 0x2, 0x2, 0x1020652, 0x1633546, ...)
            	/usr/local/go/src/reflect/value.go:460 +0x650
            reflect.Value.Call(0xc000357680, 0xc0000cb228, 0x13, 0xc000061740, 0x2, 0x2, 0x2f4d968, 0xf, 0x0)
            	/usr/local/go/src/reflect/value.go:321 +0x98
            github.com/docker/docker/internal/test/suite.Run.func2(0xc000b8de00)
            	/go/src/github.com/docker/docker/internal/test/suite/suite.go:57 +0x28c
            testing.tRunner(0xc000b8de00, 0xc0001568c0)
            	/usr/local/go/src/testing/testing.go:909 +0xd6
            created by testing.(*T).Run
            	/usr/local/go/src/testing/testing.go:960 +0x36c

=== FAIL: s390x.integration-cli TestDockerPluginSuite (177.75s)
```

Observed only on s390x and ppc64le

Still a thing; seen failing on https://ci-next.docker.com/public/blue/rest/organizations/jenkins/pipelines/moby/branches/master/runs/541/nodes/45/steps/295/log/?start=0

```
=== FAIL: ppc64le.integration-cli TestDockerPluginSuite/TestPluginInstallArgs (15.01s)
    --- FAIL: TestDockerPluginSuite/TestPluginInstallArgs (15.01s)
        suite.go:65: test suite panicked: runtime error: invalid memory address or nil pointer dereference
            goroutine 26798 [running]:
            runtime/debug.Stack(0xc0010494d0, 0x11539740, 0x12a81d00)
            	/usr/local/go/src/runtime/debug/stack.go:24 +0x8c
            github.com/docker/docker/internal/test/suite.failOnPanic(0xc000d0bd00)
            	/go/src/github.com/docker/docker/internal/test/suite/suite.go:65 +0x58
            panic(0x11539740, 0x12a81d00)
            	/usr/local/go/src/runtime/panic.go:679 +0x1bc
            github.com/docker/docker/integration-cli.(*DockerPluginSuite).TestPluginInstallArgs.func1(0xc001119da0)
            	/go/src/github.com/docker/docker/integration-cli/docker_cli_plugins_test.go:200 +0xdc
            github.com/docker/docker/testutil/fixtures/plugin.CreateInRegistry(0x11c67660, 0xc001119d40, 0xc000f121e0, 0x2f, 0x0, 0xc0010499f0, 0x1, 0x1, 0x0, 0x0)
            	/go/src/github.com/docker/docker/testutil/fixtures/plugin/plugin.go:95 +0x178
            github.com/docker/docker/integration-cli.(*DockerPluginSuite).TestPluginInstallArgs(0xc00090f010, 0xc000d0bd00)
            	/go/src/github.com/docker/docker/integration-cli/docker_cli_plugins_test.go:199 +0x140
            reflect.Value.call(0xc000d03320, 0xc000738390, 0x13, 0x117bdf2b, 0x4, 0xc0009cbf08, 0x2, 0x2, 0x1058f93c, 0x8000, ...)
            	/usr/local/go/src/reflect/value.go:460 +0x4f4
            reflect.Value.Call(0xc000d03320, 0xc000738390, 0x13, 0xc0009d5708, 0x2, 0x2, 0x0, 0x0, 0x0)
            	/usr/local/go/src/reflect/value.go:321 +0x90
            github.com/docker/docker/internal/test/suite.Run.func2(0xc000d0bd00)
            	/go/src/github.com/docker/docker/internal/test/suite/suite.go:57 +0x230
            testing.tRunner(0xc000d0bd00, 0xc00010e9a0)
            	/usr/local/go/src/testing/testing.go:909 +0xc8
            created by testing.(*T).Run
            	/usr/local/go/src/testing/testing.go:960 +0x2e8
```

--- a/testutil/fixtures/plugin/plugin.go
+++ b/testutil/fixtures/plugin/plugin.go
@@ -91,6 +91,7 @@ func CreateInRegistry(ctx context.Context, repo string, auth *types.AuthConfig, 
 	}
 
 	var cfg Config
+	cfg.PluginConfig = &types.PluginConfig{}
 	for _, o := range opts {
 		o(&cfg)
 	}

